tests/rados/test_norecovery_flag_functionality.py

```diff
@@ -0,0 +1,81 @@
+"""
+This file contains the  methods to verify the  norecover flag.
```

Comment:
Please add the bugzilla IDs for which the module is written to test.

Reply:
Added the details.